src/Middleware/SpaServices.Extensions/src/Microsoft.AspNetCore.SpaServices.Extensions.csproj

```diff
@@ -15,4 +15,11 @@
     <InternalsVisibleTo Include="Microsoft.AspNetCode.SpaServices.Extensions.Tests" />
   </ItemGroup>
 
+  <ItemGroup Condition=" '$(AspNetCoreMajorMinorVersion)' == '5.0' ">
+    <!--
+      Dependency (now in shared Fx and a transitive ref for netstandard2.x) was removed in 5.0. Suppression can be
```

Comment:
Thanks for taking care of the unintentional conflict between my #25227 and @pranavkm's #25290. But, this comment is incorrect. Should just say the Microsoft.AspNetCore.SpaServices package doesn't exist anymore, I think. Please do a follow up PR so that we don't get confused down the road.

Reply:
Done in commit https://github.com/dotnet/aspnetcore/commit/44175edd79d385e7b138d00da99a883125e0dcda

Reply:
Thanks @JunTaoLuo. What PR was / is that commit part of❔